spec/index/categorical_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper.rb'
+
+describe Daru::CategoricalIndex do
+  let(:idx1) { Daru::CategoricalIndex.new [:a, :b, :a, :a, :c] }
+  let(:idx2) { Daru::CategoricalIndex.new [0, 1, 0, 0, 2] }
```

Comment:
a) informative names! something like `symbols_idx` and `numbers_idx`
b) under that common describe, you can use `described_class.new` instead of `Daru::CategoricalIndex.new` (sometimes it looks cleaner)